src/app/components/team/Newsletter/NewsletterScheduler.js

```diff
@@ -59,7 +66,9 @@ const NewsletterScheduler = ({
             }
             arrow
           >
-            <InfoOutlinedIcon className={styles['tooltip-icon']} />
+            <span>
```

Comment:
note to reviewers, not sure why, but I had to wrap the icon in a span when I switched it to a custom svg. I think it might have to do with the toolitp component, but 🤷

Reply:
Yeah I'm guessing it's a quirk of `Tooltip` -- it's [currently just wrapping the material-ui tooltip](https://github.com/meedan/check-web/blob/406041bebc3f1c27ef50aca2658f3653d52256ed/src/app/components/cds/alerts-and-prompts/Tooltip.js) so it's probably expecting a material-ui style span wrapper around the icon.